In [2]:
import pandas as pd
import os

In [4]:
df = pd.read_excel('/nfs/masi/MCL/file/clinical/LabelFile/Vandy3/VANDY3MRNs3.10 2020-08-12 ppm.xlsx')
#df = df[df['Histologic Type'] == df['Histologic Type']]
vandy3_list = df['MCL_ID'].tolist()
vandy3_list = [str(i) for i in vandy3_list]
print (len(set(vandy3_list)))


644


In [6]:
data_root = '/nfs/masi/MCL/nifti/combine'
xnat_list = os.listdir(data_root)

print (len(set(xnat_list) & set(vandy3_list)))
xnat = []
for subj in xnat_list:
    
    if len(os.listdir(data_root + '/' + subj)) > 1: 
        
        xnat.append(subj)
        


195


In [16]:
len(set(xnat) & set(vandy3_list))


72

# update at 20200825

| Total Subject | 644 |

| With Image    | 195 | 

| With repreated Image | 72 | 
